### PR TITLE
build: avoid CVE via .cveignore file

### PR DESCRIPTION
**Pull request by padamstx**

### CODE REVIEW

File: scripts/create-binaries.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (pyrooka) [inline scripts/create-binaries.sh:1]: Why is this change? I feel it's somewhat specific to the Travis' Ubuntu image...:)

**Review by pyrooka — APPROVED** [review]
LGTM!

File: scripts/create-binaries.sh (hunk 1 above)
Comment (padamstx) [reply scripts/create-binaries.sh:1]: I made it consistent with the other build script (deploy-container-image.sh).  My understanding is that using `#!/usr/bin/env bash` is "better" in some respects as it doesn't depend on a hard-coded location for the `bash` executable.

**Pull request merged**